### PR TITLE
Changes 5.56mm to 5.45mm, now 20% less stupid

### PR DESCRIPTION
**Pull request by SpadesNeil**
The same as https://github.com/PolarisSS13/Polaris/pull/3612 except this time I'm not a dumbass and accidentally overwriting other changes because the Github client decided not to sync with Polaris.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -111,7 +111,6 @@
 	force = 10
 	caliber = "7.62mm"
```
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Caliber still doesn't seem to match, but I don't know what it actually does anymore.

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 2]
```diff
@@ -181,14 +180,13 @@
 	force = 10
 	slot_flags = 0
 	max_shells = 50
-	caliber = "5.56mm"
+	caliber = "5.45mm"
 	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 1, TECH_ILLEGAL = 2)
 	slot_flags = SLOT_BACK
-	ammo_type = "/obj/item/ammo_casing/a556" // Is this really needed anymore?
```
Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/automatic.dm]: As mentioned above, not sure what precisely this does, but I'm not sure why it's being removed.

Comment (SpadesNeil) [inline code/modules/projectiles/guns/projectile/automatic.dm]: This _used_ to be code used to fill up the gun with the correct kind of bullets when it spawned. This is completely useless now, as this gun uses a magazine, which already has bullets spawned inside of it.

Comment (Anewbe) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Alrighty.

**Pull request merged**